src/main/webapp/script.js

```diff
@@ -110,6 +110,719 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+/** Gets the ID of the YouTube video that the user inputs. */
+function getId() {
+  const liveStreamLink = document.getElementById('live-stream-link').value;
+  return getIdFromUrl(liveStreamLink);
+}
+
+/** Gets the ID of the YouTube video that the user inputs. */
+function storeLiveStreamInfo() {
+  const liveStreamLink = document.getElementById('live-stream-link').value;
+  return false;
+}
+
+/** Gets the ID of a YouTube video from its URL.
+    Will work only for a URL of a certain format
+    (i.e. youtube.com/watch?v=). */
+function getIdFromUrl(url) {
+  return url.substring(url.lastIndexOf('=') + 1);
+}
+
+/** Videos: List Retrieval */
+// https://apis.google.com/js/api.js
```

Comment:
The variables used in this function and the function itself is not readable as is. Have you looked at ways of directly including the api.js file script instead of having to copy the contents here?

Reply:
I tried just incorporating the api.js file with a <script> tag, but it didn't work because of how the auth loads (i.e. the content needs to be triggered directly by a function from script.js); in any case, I have to set the API and OAuth keys, so I need the functions there anyways.